test/helper.rb

```diff
@@ -19,6 +26,12 @@ def samples_path
 end
 
 def sample_blob(name)
-  name = File.join(samples_path, name) unless name =~ /^\//
-  Linguist::FileBlob.new(name, samples_path)
+  filepath = (name =~ /^\//)? name : File.join(samples_path, name)
+  Linguist::FileBlob.new(filepath, samples_path)
+end
```

Comment:
> Is this still used anywhere?  

Yep, a bunch. For example: https://github.com/github/linguist/blob/b3ff84872be787551556d144ee4ef133cf8b1a33/test/test_file_blob.rb#L34-L37

> It took me a while to parse the difference between

Yeah, it might be worth trying to rename these at some point.